### ISSUE TITLE
Convert to PWA

### ISSUE THREAD

**Issue by nrgapple** [monicahq/monica#4499]
You should update this into a PWA so that the mobile client is more friendly

**Comment by GitJamz** [comment]
Yes.. Please.. since there seem to be no progress on mobile application PWA is a must..

**Comment by GitJamz** [comment]
Comeon this project is semi-dead? Please let us donate for this feature then.. We need it!

**Comment by MayorMonty** [comment]
Is there any progress on this? The client itself is actually pretty responsive, so I think a PWA would be the best approach for a mobile solution. 

To make the app [installable](https://web.dev/install-criteria/), we'll need:

- [ ] Manifest File
- [ ] Service Worker, and we'll need to include a valid offline fallback page

Additionally, some niceties to include would be to integrate the notification system with Web Push, through the service worker. Of course, PWAs traditionally have pretty terrible support on iOS. 

If there's not any motion on this, I can spearhead?

**Comment by asbiin** [comment]
@GitJamz the project is not dead at all...

**Comment by Casuallynoted** [comment]
Thought it would be worth adding that since this issue was made, iOS has had increasingly better support for PWAs, including very good web push support. Would also be in favor of something like this.
